### DISCUSSION TITLE
posts/2022/fc697ca8/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#10] [Announcements]
# posts/2022/fc697ca8/

强网杯 2022 Misc 谍影重重 writeup

https://blog.gztime.cc/posts/2022/fc697ca8/

**Comment by ghost** [comment]
压缩包密码是 md5(b"api.ipify.org") = "08229f4052dde89671134f1784bed2d6" 是怎么推理出来的？

**Reply by GZTimeWalker** [reply]
压缩包中有注释：`Password is md5(api address) :)`
